src/index.mjs

```diff
@@ -3,6 +3,7 @@
 import './patch.mjs';
 
 import { $, stdin, argv, question } from "zx";
+import { existsSync } from "fs";
```

Comment:
Let's use the async version of this! We already have `doesDirectoryExist` in `utils.mjs` but it works for files too. Perhaps we can just rename it to `doesPathExist` and use it here.